core/SConscript.unix

```diff
@@ -83,6 +83,7 @@ CPPDEFINES_MOD += [
     ('USE_CARDANO', '1' if EVERYTHING else '0'),
     ('USE_NEM', '1' if (EVERYTHING and TREZOR_MODEL == "T2T1") else '0'),
     ('USE_EOS', '1' if (EVERYTHING and TREZOR_MODEL == "T2T1") else '0'),
+    ('MICROPY_ROM_TEXT_COMPRESSION', '1'),
```

Comment:
Can we put this in `mpconfigport.h` along the other `MICROPY_` defines?

Reply:
841f7c8d75

Reply:
Squashed (after a rebase over #5037).